docs/man/docker-create.1.md

```diff
@@ -57,6 +58,9 @@ IMAGE [COMMAND] [ARG...]
 **--add-host**=[]
    Add a custom host-to-IP mapping (host:ip)
 
+**--blkio-weight**=0
+   Block IO weight (relative weight) accepts a weight value between 10 to 1000.
```

Comment:
between 10 *and* 1000

(To be fully correct, "between 9 and 1001", but I don't think that will help anyone)